### ISSUE TITLE
Test version v0.1.0

### ISSUE THREAD

**Issue by webschik** [webschik/typescript-polyfills-generator#1]
Hi @RehanSaeed, @Ciantic!

I remember that you were interested in solving the issue https://github.com/Microsoft/TypeScript/issues/20095.

I've created this module as a Proof of Concept and if you have time and interest in this feature, please try it and give me your feedback.

**Comment by timocov** [comment]
Hey @webschik, thank you for your work!

What do you think about using TypeScript API to get actual types to reduce false-positive cases? For example, right now call `ctx.fill()` where `ctx` is `CanvasRenderingContext2D` might add import `core-js/modules/es6.array.fill.js`.

Some time ago I've looking into the same problem (creating custom transformer to generate polyfills) and may share my insights with you if you wish.

**Comment by webschik** [comment]
@timocov, definitely `TypeChecker` will help to reduce false-positive cases, but it's not available in  [ts-loader custom transformers](https://github.com/TypeStrong/ts-loader#getcustomtransformers--program-program---before-transformerfactory-after-transformerfactory--)  and this module is built around this API.

I think it will be fixed only when TypeScript will support custom transformers out of the box - https://github.com/microsoft/TypeScript/issues/14419

**Comment by timocov** [comment]
> it's not available in ts-loader custom transformers 

But you can get it from passed `program` instance, can't you?

**Comment by webschik** [comment]
You're right, I remember that `program` was not available if you pass `getCustomTransformers` as a string (path to file), but now it seems working. 

I will investigate it more later.
Thanks!

**Comment by ViieeS** [comment]
@webschik Hi! Nice work and great module! I am surprised there is no such functionality in ts-loader as well as confused about `target` and `lib` compiler options.

Any plans for further support and v1 release?
